api/src/pcapi/core/offers/offer_metadata.py

```diff
@@ -60,6 +61,14 @@ def _get_event_metadata_from_offer(offer: offers_models.Offer) -> Metadata:
         firstBeginningDatetime: datetime.datetime = offer.firstBeginningDatetime
         event_metadata["startDate"] = firstBeginningDatetime.isoformat(timespec="minutes")
 
+    if offer.isDigital:
+        event_metadata["eventAttendanceMode"] = "OnlineEventAttendanceMode"
+    else:
+        event_metadata["eventAttendanceMode"] = "OfflineEventAttendanceMode"
```

Comment:
je ne sais pas si c'est bien, j'aurais plutot écrit avec une ternaire

```suggestion
    event_metadata["eventAttendanceMode"] = "OnlineEventAttendanceMode" if offer.isDigital else "OfflineEventAttendanceMode"
```